dlp/snippets/inspect/inspect_test.go

```diff
@@ -681,3 +688,207 @@ func TestInspectTableWithCustomHotword(t *testing.T) {
 		t.Errorf("TestInspectTableWithCustomHotword got %q, want %q", got, want)
 	}
 }
+
+func TestInspectDataStoreSendToScc(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	var buf bytes.Buffer
+	u := uuid.New().String()[:8]
+	datastoreNamespace := fmt.Sprint("golang-samples" + u)
+	datastoreKind := "task"
+
+	if err := inspectDataStoreSendToScc(&buf, tc.ProjectID, datastoreNamespace, datastoreKind); err != nil {
+		t.Fatal(err)
+	}
+
+	got := buf.String()
+	if want := "Job created successfully:"; !strings.Contains(got, want) {
+		t.Errorf("InspectBigQuerySendToScc got %q, want %q", got, want)
+	}
+}
+
+func TestInspectWithStoredInfotype(t *testing.T) {
```

Comment:
issue: there should only be one test per file.

See https://googlecloudplatform.github.io/samples-style-guide/#dedicated-testing-per-sample .

Reply:
Thank you @telpirion for your review, We can definitely create a separate test file for each test case, 
Before proceeding, could you please clarify whether this change is specific to the `inspect` package only My concern is that it might lead to inconsistency in the codebase of DLP. apart from that, I've gone through the sample guide link you shared, and while it emphasizes having each test case cover a single sample, I couldn't find explicit guidance stating that there should be only one test per file. However, please do let me know if there's anything I might have overlooked.

Reply:
Hey @telpirion, For this sample add a test in a new file separate file.